hw/syn/data/dc.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tool_srcs: ["{proj_root}/hw/foundry/syn/{tool}/setup.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/start-dc.sh"
```

Comment:
Wildcard is supported here BTW. For example: `"{proj_root}/hw/syn/tools/{tool}/*"`

Reply:
good to know, thanks. I believe there are more scripts here which should not necessarily be copied over, so individual copying is fine here.